evap/evaluation/tests/test_commands.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+from django.utils.six import StringIO
+
+from django.core import management, mail
+from unittest.mock import patch
+from django.test.testcases import TestCase
+from django.test.utils import override_settings
+from model_mommy import mommy
+
+from evap.evaluation.models import UserProfile, Course
+
+
+class TestRefreshResultsCacheCommand(TestCase):
+    def test_calls_calculate_results(self):
+        with patch('evap.evaluation.tools.calculate_results') as mock:
+            management.call_command('refresh_results_cache', stdout=StringIO())
+
+        self.assertEqual(mock.call_count, Course.objects.count())
```

Comment:
Probably. I am not so sure about this test anyway. The Command basically consists of one for loop. Should we test this at all? I just added it for coverage. in case python 4 comes along :)